bundle/tests/environment_git_test.go

```diff
@@ -5,18 +5,19 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/databricks/cli/bundle/config/mutator"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestGitAutoLoadWithEnvironment(t *testing.T) {
-	b := load(t, "./environments_autoload_git")
+	b := load(t, "./environments_autoload_git", mutator.LoadGitDetails())
```

Comment:
You can keep `load` as-is and just do a separate call to `bundle.Apply(b, mutator.LoadGitDetails)`

Reply:
bundle.Apply also takes the context which in this case is hidden inside load().

Reply:
I prefer not to break the context but I see that other tests are written like that at the moment, so I've updated my tests according to your suggestion.